resources/views/components/dashboards/crew.blade.php

```diff
@@ -48,49 +54,139 @@
                         :route="'moderator.list'"
                         :param="'users'"
-                        :roleColour="'crew'"/>
+                        :roleColour="'crew'"
+                        :styleMode="'alternate'"/>
                     <x-dashboards.blocks.crew
                         :label="'Booths'"
                         :count="Booth::where('is_approved', 1)->count()"
                         :icon="'M13.5 21v-7.5a.75.75 0 01.75-.75h3a.75.75 0 01.75.75V21m-4.5 0H2.36m11.14 0H18m0 0h3.64m-1.39 0V9.349m-16.5 11.65V9.35m0 0a3.001 3.001 0 003.75-.615A2.993 2.993 0 009.75 9.75c.896 0 1.7-.393 2.25-1.016a2.993 2.993 0 002.25 1.016c.896 0 1.7-.393 2.25-1.016a3.001 3.001 0 003.75.614m-16.5 0a3.004 3.004 0 01-.621-4.72L4.318 3.44A1.5 1.5 0 015.378 3h13.243a1.5 1.5 0 011.06.44l1.19 1.189a3 3 0 01-.621 4.72m-13.5 8.65h3.75a.75.75 0 00.75-.75V13.5a.75.75 0 00-.75-.75H6.75a.75.75 0 00-.75.75v3.75c0 .415.336.75.75.75z'"
                         :route="'moderator.booths.index'"
                         :param="'booths'"
-                        :roleColour="'crew'"/>
+                        :roleColour="'crew'"
+                        :styleMode="'alternate'"/>
                     <x-dashboards.blocks.crew
                         :label="'Presentations'"
                         :count="$numberOfScheduledPresentations"
                         :icon="'M3.75 3v11.25A2.25 2.25 0 006 16.5h2.25M3.75 3h-1.5m1.5 0h16.5m0 0h1.5m-1.5 0v11.25A2.25 2.25 0 0118 16.5h-2.25m-7.5 0h7.5m-7.5 0l-1 3m8.5-3l1 3m0 0l.5 1.5m-.5-1.5h-9.5m0 0l-.5 1.5m.75-9l3-3 2.148 2.148A12.061 12.061 0 0116.5 7.605'"
                         :route="'moderator.presentations.index'"
-                        :roleColour="'crew'"/>
+                        :roleColour="'crew'"
+                        :styleMode="'alternate'"/>
                     <x-dashboards.blocks.crew
                         :label="'Companies'"
                         :count="Company::where('is_approved', 1)->count()"
                         :icon="'M3.75 21h16.5M4.5 3h15M5.25 3v18m13.5-18v18M9 6.75h1.5m-1.5 3h1.5m-1.5 3h1.5m3-6H15m-1.5 3H15m-1.5 3H15M9 21v-3.375c0-.621.504-1.125 1.125-1.125h3.75c.621 0 1.125.504 1.125 1.125V21'"
                         :route="'moderator.companies.index'"
-                        :roleColour="'crew'"/>
+                        :roleColour="'crew'"
+                        :styleMode="'alternate'"/>
                 </dl>
             </div>
         </div>
     </div>
+    <div class="py-4 px-2 mx-auto max-w-7xl">
+        <div>
+            <h3 class="leading-6 font-semibold text-xl dark:text-white">Increase graph speakers, booths, presentations, companies</h3>
+            <div class="pt-6 px-6 pb-12 rounded-lg overflow-hidden relative">
+                <div class="chart-container" style="position: relative; height:40vh; width:73vw">
+                    <canvas id="statsChart"></canvas>
+                </div>
+
+                @push('scripts')
```

Comment:
Please separate that JS from the file, move it to a js file in `public` and reference it here or in the layout